apps/coscrad-frontend-e2e/src/support/commands.ts

```diff
@@ -99,11 +99,13 @@ Cypress.Commands.add('login', () => {
     // TODO Split all chains on top of click and fix the lint error
     cy.get('#username').click();
 
-    cy.get('#username').type(Cypress.env('username'));
+    // we don't want this sensitive info showing up in CI logs, for example
+    cy.get('#username').type(Cypress.env('username'), { log: false });
```

Comment:
Nice!

Reply:
Yea. I thought I had done this before. I wonder if there's a log scraper that can alert us of suspected leaks...